docs/adr/0005-use-better-npm-audit.md

```diff
@@ -0,0 +1,77 @@
+# Use better-npm-audit for checking deps for security vulnerabilities
+
+* Status: accepted
+* Deciders: platui
+* Date: 2022-05-16
+* Components: platui maintained nodejs projects
+
+Technical Story: PLATUI-1691
+
+## Context
+
+We need to keep our libraries safe by regularly checking our dependencies for new publicly disclosed vulnerabilities.
+
+We've automated this by periodically running "npm audit" on our nodejs projects. npm audit will compare our list of dependencies against public security disclosure lists and show which of our dependencies contain vulnerabilities.
+
+We review the results of this weekly, and we have a dashboard that shows us an overview of the latest checks that have been run by repository. Any vulnerabilities found cause the repository to be displayed in red on the dashboard.
+
+In our team calendar we have a recurring prompt to review the dashboard and look for failures that need our attention and then try to update our vulnerable dependencies.
+
+Some issues surfaced by npm audit we can't resolve by updating our dependencies. However, we may find that our use case for the dependency isn't one that's impacted by the vulnerability. In that case, although we are still using a vulnerable version, we're safe to keep using it in the current way.
+
+### Problem statement
+
+Over time the number of issues we aren't able to fix has increased, and resulted in our dashboard showing all checks in red all the time. This makes it hard to notice new security vulnerabilities and means we spend time rechecking previous vulnerabilities.
+
+We would like the dashboard entries to not be displayed in red because of known issues that have already been reviewed, so that we can easily see where there are new vulnerabilities and avoid wasting time re-checking the same issues.
+
+## Decision Drivers
+
+* we would like to make a small change, so keep our current process, but improve the mechanism
+* we only want to check nodejs dependencies at the moment
```

Comment:
Is it worth adding in here that there are some options that are not currently supported by our projects, such as Dependabot?